src/modules/0.0.21/iasql_functions/rpcs/iasql_apply.ts

```diff
@@ -0,0 +1,36 @@
+import { snakeCase } from 'typeorm/util/StringUtils';
+
+import { IasqlFunctions } from '..';
+import { Context, RpcBase, RpcResponseObject } from '../../../interfaces';
+import * as iasql from '../iasql';
+
+export class IasqlApply extends RpcBase {
+  module: IasqlFunctions;
+  name = 'iasql_apply';
```

Comment:
Should we try to auto-derive this `name` variable from the class name using the `snakeCase` function?